routers/router.go

```diff
@@ -61,6 +61,7 @@ func initAPI() {
 	beego.Router("/api/acs", &controllers.ApiController{}, "POST:HandleSamlLogin")
 	beego.Router("/api/saml/metadata", &controllers.ApiController{}, "GET:GetSamlMeta")
 	beego.Router("/api/webhook", &controllers.ApiController{}, "POST:HandleOfficialAccountEvent")
+	beego.Router("/api/webhook", &controllers.ApiController{}, "GET:WXOfficialAccountToken")
```

Comment:
Don't use the "/api/webhook" name again

Reply:
This is because the interface configuration of the WeChat official account and the event reception are all the same URL, but the request type is different. In the original code, /api/webhook is already receiving events, so I use